projects/js-packages/publicize-components/src/social-store/types.ts

```diff
@@ -7,7 +7,7 @@ export type SharesData = {
 	shared_posts_count: number;
 };
 
-export type ConnectionStatus = 'ok' | 'must_reauth' | 'invalid' | 'broken' | 'refresh-failed';
+export type ConnectionStatus = 'ok' | 'broken';
```

Comment:
I know it doesn't matter as this is compile time, but the old code could end up encountering these other statuses

Reply:
Doesn't it come from the `status` property in the server responses? It's maybe not an issue!